phnt/include/ntexapi.h

```diff
@@ -721,6 +721,7 @@ NTSTATUS
 NTAPI
 NtCreateIRTimer(
     _Out_ PHANDLE TimerHandle,
+    _In_ LPVOID Reserved,
```

Comment:
* LPVOID needs to be PVOID
* `PVOID Reserved` would be `HANDLE IRTimerHandle` on Windows 10 and 11?